keyboards/kisakeyluxury/qtz/readme.md

```diff
@@ -0,0 +1,15 @@
+# qtz
+
+35% keyboard designed by Iceage
+
+* Keyboard Maintainer: [kisakey-luxury](https://github.com/kisakey-luxury)
+* Hardware Supported: *not released yet*
+* Hardware Availability: *not released yet*
+
+Make example for this keyboard (after setting up your build environment):
+
+    make qtz:default
+
+Flashing example for this keyboard:
+
+    make qtz:default:flash
```

Comment:
```suggestion
    make kisakeyluxury/qtz:default

Flashing example for this keyboard:

    make kisakeyluxury/qtz:default:flash
```

Also, 
Could you add instructions on how to enter the bootloader, to flash new firmware?  This is needed, as a new user may not be aware of how to enter to bootloader to flash new firmware.

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```